src/pages/index.js

```diff
@@ -1,11 +1,22 @@
 import 'regenerator-runtime/runtime';
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 
 import { Landing } from './unAuth/index';
 import { Home } from './auth/home';
 
 export function IndexPage({ isSignedIn }) {
   const [showAdmin, setShowAdmin] = useState(false);
+  useEffect(() => {
+    const search = window.location.search;
+    const params = new URLSearchParams(search);
+    const community = params.get('community');
+    const vertical = params.get('vertical');
+    console.log(vertical);
+    if (community && vertical) {
+      localStorage.setItem('community-name', community);
+      localStorage.setItem('community-vertical', vertical);
+    }
```

Comment:
why are we storing data in localstorage? why don't we use db, or may be we can add redux